append_lifecycle.go

```diff
@@ -44,6 +47,41 @@ const (
 	DefaultPushbackMaxOutstanding = 4096
 )
 
+var (
+	appenderAddsTotal    metric.Int64Counter
+	appenderAddHistogram metric.Int64Histogram
+	appenderTreeSize     metric.Int64Gauge
+)
+
+func init() {
+	var err error
+
+	appenderAddsTotal, err = meter.Int64Counter(
+		"appender.add.calls",
```

Comment:
I suspect we should use `tessera.*` as a namespace. This:
a) makes it easy for personalities operators to find all of the metrics we are exporting
b) avoids conflict on the remote chance that they are tracking how many times they, e.g. call the appender.

Reply:
Or to go full [guidelines](https://opentelemetry.io/docs/specs/semconv/general/naming/#recommendations-for-application-developers) (and a bit Java): `dev.transparency.tessera.*`. Seems a bit much for my tastes, but them's the recommendations.